### PR TITLE
Feat/issuer template

### PR DESCRIPTION
**Pull request by umut0**
Create one Issuer HTML for both the public and private issuer view.

Edit: Also fixed a grid layout issue in this PR. sorry for the mess

### CODE REVIEW

File: src/app/components/datatable-badges.component.ts
Comment (jona159) [inline src/app/components/datatable-badges.component.ts:39]: I think this should be ```tw-break-all```

**Review by jona159 — APPROVED** [review]

**Comment by jona159** [comment]
Can be merged after comment and conflicts are resolved

**Comment by umut0** [comment]
I get an error now if i am not logged in and try to view in institution. DO not merge yet.

**Pull request merged**
